core/app/components/StageSelect/StageSelectClient.tsx

```diff
@@ -24,16 +23,16 @@ export function StageSelectClient(props: Props) {
 					</div>
 					<Select
 						{...field}
-						value={props.stage?.id}
 						onValueChange={(value) => {
 							field.onChange(value);
 						}}
+						defaultValue={field.value}
```

Comment:
I believe we should use the value from the form rather than passing a value in

Reply:
makes sense to me!